packages/components/pie-toggle-switch/README.md

```diff
@@ -59,8 +59,9 @@ import { PieToggleSwitch } from '@justeattakeaway/pie-toggle-switch/dist/react';
 
 | Property | Type | Default | Description |
 |---|---|---|---|
-| isChecked | `Boolean` | false | Indicates whether the switch is on or off |
+| isChecked  | `Boolean` | false | Indicates whether the switch is on or off |
 | isDisabled | `Boolean` | false | Indicates whether the switch is disabled or not |
+| label      | `Object`  | `undefined` | Provides a label for the component. You can set the label position using options: `leading` | `trailing` e.g object: `label: { text: 'Label', position: 'leading' }` |
```

Comment:
Seems like the default value is incorrect?

https://github.com/justeattakeaway/pie/pull/705/files#diff-4c058adb36e0f8491d5e07aec4d39d31af1d2bac947533eb1b865b7ea54c3869R51